sources/api/prairiedog/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+//! The prairiedog library contains serializer and deserialzier implementations for kernel boot config files
+
+#![deny(rust_2018_idioms)]
+#[cfg(bottlerocket_platform = "metal")]
```

Comment:
Hm - this works for the short term, but newer AWS/VMware variants may also have the ability to use this (since they'll be built with a newer grub)...